cmd/preflight/cmd/check_operator.go

```diff
@@ -52,6 +52,12 @@ func checkOperatorCmd(runpreflight runPreflight) *cobra.Command {
 		"If empty, the default operator channel in bundle's annotations file is used.. (env: PFLT_CHANNEL)")
 	_ = viper.BindPFlag("channel", checkOperatorCmd.Flags().Lookup("channel"))
 
+	checkOperatorCmd.Flags().Duration("csv-timeout", 0, "The Duration of time to wait for the ClusterServiceVersion to become healthy.\n"+
```

Comment:
nit: " .. time in seconds .." but its implied by the 180s default value mentioned 👍

Reply:
I took out the mention of the unit here, since I switched to Duration, this could be any unit, seconds, minutes, hours, etc.

Reply:
Ah so the unit suffix is needed if not ‘—csv-timeout=300’ would mean nanoseconds

Reply:
No the suffix is required, otherwise cobra throws an error. `—csv-timeout=300` is not valid.